rvip/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>rvip</name>
+  <version>0.0.1</version>
+  <description>The rvip package</description>
+  <author>Bey Hao Yun</author>
+  <maintainer email="beyhy@artc.a-star.edu.sg">Bey Hao Yun</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>cv_bridge</depend>
+  <depend>geometry_msgs</depend>
+  <depend>image_transport</depend>
+  <depend>pcl_ros</depend>
+  <depend>rviz</depend>
+  <depend>rospy</depend>
```

Comment:
`rviz` is probably only a `exec_depend`, as we don't have any code depending on it. Only `.launch` files.

Same for `rospy`: you're not linking against it, so it cannot be a `depend`. It should be an `exec_depend`.

I believe this is one of the causes of the current CI failures.

Reply:
Addressed in [this commit](https://github.com/ros-industrial/rvip/pull/1/commits/66c190d10667e5a24df7a99a79497c7bd09d05b5).